src/lib/index.js

```diff
@@ -8,6 +8,7 @@
 export { DepositApiClient } from './DepositApiClient';
 export { DepositController } from './DepositController';
 export { DepositFormApp } from './DepositFormApp';
+export { DepositFormFeedback } from './DepositFormFeedback';
```

Comment:
See my comment for moving this to `components` and maybe rename it to `FormFeedback`